.gitignore

```diff
@@ -1,2 +1,3 @@
 *.swp
 result*
+.DS_Store
```

Comment:
nit: would prefer this to be in a separate commit. (FWIW, I personally add this to my global `~/.config/git/ignore` with Home Manager so I don’t have to worry about it.)

Reply:
TIL Git has a global ignore file